src/styles/body-blocks/products-list.scss

```diff
@@ -0,0 +1,31 @@
+.products-list {
+  display: grid;
+  grid-template: 1fr 1fr 1fr 1fr / auto auto auto auto;
```

Comment:
I think you changed order for columns and rows, that affects your tests. 
`grid-template: grid-rows  / grid-columns`
Because in your page, there are two unnecessary rows.
[here you can read more](https://developer.mozilla.org/en-US/docs/Web/CSS/grid-template)